tests/unit_tests/data_steward/retraction/retract_utils_test.py

```diff
@@ -172,6 +172,39 @@ def test_get_mapping_tables(self):
             ru.get_mapping_tables(common.EXT,
                                   self.mapping_tables + self.other_tables), [])
 
+    def test_is_combined_dataset(self):
```

Comment:
Adding a suggested test for the `get_dataset_list` function.  All it's checks fail.  Let me know if you think the function usage is inconsistent with the documentation design.  See the attached patch file.

[get_datasets_list_util_patch.txt](https://github.com/all-of-us/curation/files/5662502/get_datasets_list_util_patch.txt)

Reply:
Thanks, this is very helpful!